Rubberduck.VBEEditor/Events/VbeEvents.cs

```diff
@@ -73,6 +73,7 @@ private void RegisterComponents(string projectId, string projectName)
                 if (item.ProjectId == projectId && item.Name == projectName)
                 {
                     project = item;
+                    item.Dispose();
```

Comment:
this feels weird. Is that a value copy or a reference copy? Or rather: if we Dispose `item` is `project` also disposed?

Reply:
I agree - I'll double check that.

Reply:
That's definitely a reference copy. AFAICT this will break component registration.

Reply:
I think that was a false positive and I did not check this carefully; I've removed the unneeded `Dispose`.